src/components/Header.vue

```diff
@@ -34,8 +34,11 @@ export default {
   },
   methods: {
     pressedEnter() {
-      this.$emit('filter-user', parseInt(this.userId))
-      this.uidInTitle = this.userId
+      let u = parseInt(this.userId)
```

Comment:
hmmm partInt is iffy because if my input is  '1241sdafs', parseInt returns 1241. I do think we should validate if u.length != 0 however. Just a thought

Reply:
I think in header
rules="[(v) => v <= 11043 || 'Select ID from 0 to 11043']" 
takes care of the fact that no alphabets or any other symbols are allowed.